### ISSUE TITLE
Object.setPrototypeOf in IE 10

### ISSUE THREAD

**Issue by christopherjbaker** [feathersjs/feathers-errors#70]
We are running into an error using `Object.setPrototypeOf` in IE10, canjs/can-connect-feathers#65. 
The calls to this are being added in your compile phase by babel. I did notice that you are using https://www.npmjs.com/package/babel-plugin-transform-builtin-extend but have not enabled approximate mode, as required for IE <= 10. Recompiling with approximate mode does cause our tests to pass, though it causes 2 of yours to fail: [returns NotFound error](https://github.com/feathersjs/feathers-errors/blob/master/test/not-found-handler.test.js#L28) and (error stack is not undefined)[https://github.com/feathersjs/feathers-errors/blob/master/test/index.test.js#L181] (the second of which seems much more involved).

### System configuration

Tell us about the applicable parts of your setup.

**Operating System**: Windows 7 and 10

**Browser Version**: IE 10

**Module Loader**: StealJS 1.0.5

**Comment by daffl** [comment]
Including `babel-polyfill` or `core-js` in the build will most likely fix the problem.

**Comment by marshallswain** [comment]
I've tried separately including both polyfills, but I was unable to fix the problem in one version of IE, but it for sure was less than IE 10.

**Comment by daffl** [comment]
I'm not sure what to do here. [According to Saucelabs](https://saucelabs.com/u/feathersjs) the test for the client build are passing in all evergreen browsers and IE 9 - 11.

**Comment by christopherjbaker** [comment]
I did some more digging. It seems the [offending code](https://github.com/feathersjs/feathers-client/blob/41dd062ee0886c0e6f4620ec1dd8d012e7bbcda2/dist/feathers.js#L1892) is not run in the browser tests. I added the following to src/browser/test.js (in feathers-client), and it failed in browserstack.

```javascript
it('can create errors', () => {
	new feathers.errors.NotAuthenticated('Not Authenticated')
});
```

**Comment by daffl** [comment]
This has been closed via #73
